src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/Arm/Sve2.cs

```diff
@@ -878,6 +878,621 @@ internal Arm64() { }
         public static Vector<ulong> ShiftLogicalRoundedSaturate(Vector<ulong> value, Vector<long> count) => ShiftLogicalRoundedSaturate(value, count);
 
 
+        // Shift right and insert
+
+        /// <summary>
+        /// svuint8_t svsri[_n_u8](svuint8_t op1, svuint8_t op2, uint64_t imm3)
+        ///   SRI Ztied1.B, Zop2.B, #imm3
+        /// </summary>
+        public static Vector<byte> ShiftRightAndInsert(Vector<byte> left, Vector<byte> right, [ConstantExpected] byte shift) => ShiftRightAndInsert(left, right, shift);
+
+        /// <summary>
+        /// svint16_t svsri[_n_s16](svint16_t op1, svint16_t op2, uint64_t imm3)
```

Comment:
>svsri [](http://example.com/codeflow?start=22&length=5)

the parameter name  `left` doesn't make sense here. `left` acts as a destination vector where results are merged. When I see `left` and `right`, I expect some kind of binary operation done by API, which is definitely not the case here. We should take this to API review committee.